Gopkg.lock

```diff
@@ -0,0 +1,301 @@
+  name = "github.com/boltdb/bolt"
+  packages = ["."]
+  revision = "4b1ebc1869ad66568b313d0dc410e2be72670dda"
+
+[[projects]]
+  name = "github.com/cespare/xxhash"
+  packages = ["."]
+  revision = "1b6d2e40c16ba0dfce5c8eac2480ad6e7394819b"
+
+[[projects]]
+  name = "github.com/davecgh/go-spew"
+  packages = ["spew"]
+  revision = "346938d642f2ec3594ed81d874461961cd0faa76"
+
+[[projects]]
+  name = "github.com/dgrijalva/jwt-go"
+  packages = ["."]
+  revision = "24c63f56522a87ec5339cc3567883f1039378fdb"
+
+[[projects]]
+  name = "github.com/dgryski/go-bits"
+  packages = ["."]
+  revision = "2ad8d707cc05b1815ce6ff2543bb5e8d8f9298ef"
+
+[[projects]]
+  name = "github.com/dgryski/go-bitstream"
+  packages = ["."]
+  revision = "7d46cd22db7004f0cceb6f7975824b560cf0e486"
+
+[[projects]]
+  name = "github.com/glycerine/go-unsnap-stream"
+  packages = ["."]
+  revision = "62a9a9eb44fd8932157b1a8ace2149eff5971af6"
+
+[[projects]]
+  name = "github.com/gogo/protobuf"
+  packages = [
+    "codec",
+    "gogoproto",
+    "proto",
+    "protoc-gen-gogo/descriptor",
+    "sortkeys",
+    "types"
+  ]
+  revision = "1c2b16bc280d6635de6c52fc1471ab962dc36ec9"
+
+[[projects]]
+  name = "github.com/golang/protobuf"
+  packages = ["proto"]
+  revision = "1e59b77b52bf8e4b449a57e6f79f21226d571845"
+
+[[projects]]
+  name = "github.com/golang/snappy"
+  packages = ["."]
+  revision = "d9eb7a3d35ec988b8585d4a0068e462c27d28380"
+
+[[projects]]
+  name = "github.com/google/go-cmp"
+  packages = [
+    "cmp",
+    "cmp/cmpopts",
+    "cmp/internal/diff",
+    "cmp/internal/value"
+  ]
+  revision = "18107e6c56edb2d51f965f7d68e59404f0daee54"
+
+[[projects]]
+  name = "github.com/influxdata/influxql"
+  packages = [
+    ".",
+    "internal"
+  ]
+  revision = "c0a433ab09b6da369995baf2ee9e5303a5ab69e4"
+
+[[projects]]
+  name = "github.com/influxdata/usage-client"
+  packages = ["v1"]
+  revision = "6d3895376368aa52a3a81d2a16e90f0f52371967"
+
+[[projects]]
+  name = "github.com/influxdata/yamux"
+  packages = ["."]
+  revision = "1f58ded512de5feabbe30b60c7d33a7a896c5f16"
+
+[[projects]]
+  name = "github.com/influxdata/yarpc"
+  packages = [
+    ".",
+    "codes",
+    "status",
+    "yarpcproto"
+  ]
+  revision = "036268cdec22b7074cd6d50cc6d7315c667063c7"
+
+[[projects]]
+  name = "github.com/jsternberg/zap-logfmt"
+  packages = ["."]
+  revision = "5ea53862c7fa897f44ae0b3004283308c0b0c9d1"
+
+[[projects]]
+  name = "github.com/jwilder/encoding"
+  packages = ["simple8b"]
+  revision = "27894731927e49b0a9023f00312be26733744815"
+
+[[projects]]
+  name = "github.com/mattn/go-isatty"
+  packages = ["."]
+  revision = "6ca4dbf54d38eea1a992b3c722a76a5d1c4cb25c"
+
+[[projects]]
+  name = "github.com/matttproud/golang_protobuf_extensions"
+  packages = ["pbutil"]
+  revision = "c12348ce28de40eed0136aa2b644d0ee0650e56c"
+
+[[projects]]
+  branch = "master"
+  name = "github.com/mschoch/smat"
+  packages = ["."]
+  revision = "90eadee771aeab36e8bf796039b8c261bebebe4f"
```

Comment:
Both github.com/mschoch/smat and github.com/willf/bitset are present here but not in Godeps. I'm not sure if that means they're a transitive dependency that gdm didn't pick up, maybe?

All the other revisions appear to match gdm at a glance.

Reply:
This one in particular seems to be a direct dependency of roaring bitmap so that is definitely strange that GDM didn't pick it up.